src/std/rpc/ws_client/events.rs

```diff
@@ -1,246 +1,263 @@
-// Copyright 2019 Parity Technologies (UK) Ltd.

```

Comment:
Do we have a line-ending issue here?

Reply:
What shall I check? I ran cargo fmt and I don't understand why it happens in this file only.

Reply:
Check out https://www.jetbrains.com/help/clion/configuring-line-endings-and-line-separators.html, maybe that can help

Reply:
To prevent further cargo fmt discrepancies, we should use a [rustfmt.toml](https://github.com/integritee-network/worker/blob/master/rustfmt.toml). This one defines the line-endings to be `Unix` style.

Reply:
But maybe you are right, and the file was wrong before. :)

Reply:
According to https://github.com/scs/substrate-api-client/pull/153?file-filters%5B%5D=.rs#discussion_r695428706, my settings are not set to unix line-endings. If I change them, all the files are affected. As the problem happened only in one file, I'm  not sure I have to do it

Reply:
Uff, it seems that you are right. We are currently not using unix-file endings. Then I suggest, we leave it as is here, and I will make a PRwith [this](https://github.com/scs/substrate-api-client/tree/cl-add-rustfmt-toml) branch, where I added the rustfmt.toml.